clients/cobol-lsp-vscode-extension/src/__tests__/MiddlewareTest.spec.ts

```diff
@@ -73,3 +73,17 @@ describe("Copybook downloader", () => {
         expect(next).toHaveBeenCalledWith(params, null);
     });
 });
+
+const each = require("jest-each").default;
+describe("Parse line parsing test", () => {
```

Comment:
I would provide a better description like: "Given an input string then ```parseLine``` returns the ```RequestLine``` expected or ```undefined```

Reply:
This description is only a part of a test case name. See an example:
![image](https://user-images.githubusercontent.com/60617592/94547635-d0948b80-024f-11eb-9f94-1077918f74bc.png)

Reply:
I saw it, so I will reformulate it like: "ParseLine() return a RequestLine object or undefined when and then the input is" + foreach.
You can skip this change if u think is already well enough explicative ;)

Reply:
What about:
```
describe("ParseLine() returns a RequestLine object or undefined in the following cases:", () => {
```
The description shows that it will be a parametrized test.

Reply:
YA! u got my point that way :)